Sources/SwiftKuery/QueryBuilder.swift

```diff
@@ -18,24 +18,57 @@ import Foundation
 
 // MARK: QueryBuilder
 
-/// Used in code dealing with variances between the various database engines. As
-/// different databases have different query syntax, sometimes changes need to be
-/// made when generating the actual SQL statement to run. Additional needed changes
-/// should be done by updating QueryBuilder substitutions array. Every query component
-/// then builds its string representation using that array.
+/**
+ Used in code dealing with variances between the various database engines. As
+ different databases have different query syntax, sometimes changes need to be
+ made when generating the actual SQL statement to run. Additional needed changes
+ should be done by updating QueryBuilder substitutions array. Every query component
```

Comment:
I think we should add some words here so that users who are using the existing database plugins realise that what you are describing here won't be relevant to them.  Perhaps something like...
"If you are adding support for a new database plugin you may need to make additions to the `QueryBuilder` substitutions array as every query component builds its string representation using that array."